### ISSUE TITLE
iegraph : color() has no effect in baroption() option

### ISSUE THREAD

**Issue by kbjarkefur** [worldbank/ietoolkit#346]
Issue submitted by email from @oscarmdiazb :

> For a while, I've been trying to change the colours of the bars when using the iegraph command, but without success. Would you mind providing an example? I've tried the variations of the following code: 
```
sysuse auto, clear 
reg price foreign 
iegraph foreign,  varlabels legend(pos(6) rows(1)) baroptions(color(green))     barlabel barlabelformat( %9.2f)
```

**Comment by kbjarkefur** [comment]
The error reproduces as submitted by user.

The fix was quite straightforward and is implemented in f4ac0a1e27ed6b5aba02904f0273e93974b8b9ec.

**Comment by kbjarkefur** [comment]
OP followed up and suggested that `baroptions()` should be possible to set colors for individual bars. This is not possible using  `baroptions()` as the bar options specified in that option is added to all bars.

Instead, in 0bfb72cb9ba7b7f0acc05a4383bd6d6ea6211440 a new option `barcolors()` was added. This option allows the user to manually specify colors, as in `` barcolors(`" red gs14 "215 25 28" gold cranberry "') ``. Any color listed in 
https://www.stata.com/manuals13/g-4colorstyle.pdf may be used.
